DataTransformation/ConditionalSplit.cs

```diff
@@ -156,6 +156,7 @@ private AsyncFeed BuildDynamicFeed(string name)
 
             if (Finished)
             {
+                Debug.Print($"Started finished stream {dynamicFeed.Name}");
```

Comment:
Started finished stream doesn't make a lot of sense to me.

Reply:
You're correct, that is confusing. Unfortunaetly I couldn't think of better nomenclature. It's an async stream, so it can be read while it's being populated (it's a queue strategy, so it's read in order. The stream has to told when it's done, so I guess we could have said "done", but I'm not sure that's any better. "Disposed" and "Closed" both seemed wrong because they conveyed different states.